dao.py

```diff
@@ -292,6 +292,29 @@ def get_tournament_by_id(self, id):
         '''id must be an ObjectId'''
         return M.Tournament.load(self.tournaments_col.find_one({'_id': id}), context='db')
 
+    def get_tournament_by_id_and_match_id(self, tournament_id, match_id):
+        tourney_m = M.Tournament.load(self.tournaments_col.find_one({'_id': tournament_id}, {'matches', 1}), context='db')
```

Comment:
not a big deal, but this should probably be called "get_match_etc" instead of "get_tournament_etc"?